scraper/.pylintrc

```diff
@@ -1,596 +0,0 @@
-[MASTER]
```

Comment:
I nuked this because we're just using the standard rules and the build system was really not wanting to run the small changes I tried to make (fail under score of 9) so I just stuck that in as an argument in `.travis.yml`